hrpc/scan.go

```diff
@@ -439,3 +477,41 @@ func RenewalScan() func(Call) error {
 		return nil
 	}
 }
+
+// ScanStatsID is an option for Scan requests to provide a ScanStatsID for the scan
+func ScanStatsID(id int64) func(Call) error {
```

Comment:
Can we include a comment here saying that this is just used internally by gohbase